### PR TITLE
Feature/stefan clem/r hashtables

### PR DESCRIPTION
**Pull request by iepoch**

### CODE REVIEW

File: resizing_hashtable/r_hashtables.py [hunk 1]
```diff
@@ -33,7 +39,17 @@ def hash(string, max):
...
 # '''
 def hash_table_insert(hash_table, key, value):
-    pass
+    index = hash(key, hash_table.capacity)
+    new_pair = LinkedPair(key, value)
+    current_pair = hash_table.storage[index]
+    while current_pair is not None and current_pair.key != key:
+        current_pair = current_pair.next
+    if current_pair is None:
+        new_pair = LinkedPair(key, value)
```
Comment (eddygonzalez9708) [inline resizing_hashtable/r_hashtables.py:56]: A new pair is already created on line 43.

Comment (iepoch) [reply resizing_hashtable/r_hashtables.py:56]: I remove it soon I don't remember why I had two.

File: resizing_hashtable/r_hashtables.py [hunk 2]
```diff
@@ -62,61 +73,95 @@ def hash_table_insert(hash_table, key, value):
...
+        # new_hash_table.storage[index] = hash_table.storage[index]
+        current_pair = hash_table.storage[x]
+        while current_pair is not None:
+            hash_table_insert(
+                new_hash_table, current_pair.key, current_pair.value)
+            current_pair = current_pair.next
+
+    return new_hash_table
 
+    new_hash = HashTable(hash_table.capacity * 2)
```
Comment (eddygonzalez9708) [inline resizing_hashtable/r_hashtables.py:140]: I believe this block of code will not get executed after the return statement on line 38.

**Review by eddygonzalez9708 — APPROVED** [review]
Great job following along with me during after hours to develop a solution for this exercise.

**Pull request merged**